kolibri/core/analytics/pskolibri/_pslinux.py

```diff
@@ -0,0 +1,309 @@
+"""Linux platform implementation."""
```

Comment:
Did you write all this? :)

Reply:
For the Linux part, most if taken from psutils. I removed the parts we didn't need and simplified what could be simplified or reused from Kolibri. Most of this file ideas are not my merit , even if it took me work to do it.